recipes/viridis/meta.yaml

```diff
@@ -0,0 +1,67 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "viridis" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "59c0598e0e144bda22fefd0174ad827d095f0ef9baf62f51334436e93b7e748b" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+    - setuptools
+  run:
+    - python
+    - numpy
+    - networkx
+
+test:
+  requires:
+    - pytest
+    - coverage
+    - pytest-cov
+  commands:
+    - py.test
+
+about:
+  home: https://github.com/jni/viridis
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: BSD 3-clause
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See http://conda.pydata.org/docs/building/meta-yaml.html#license-file
+  license_file: license.txt
```

Comment:
> it's missing the license.txt... I'll need to fix that with a new point release.

If the license file does not exist in the source code you can leave it out here for now. (And since you are the author we know you are on top of this to fix it :smile:)